test/test-helpers.js

```diff
@@ -104,6 +105,11 @@ export class MockTextTrack {
   }
 }
 
+// return an absolute version of a page-relative URL
+export const absoluteUrl = function(relativeUrl) {
+  return URLToolkit.buildAbsoluteURL(window.location.href, relativeUrl);
```

Comment:
is this compatible with our implementation?

Reply:
I believe yes for two reasons:

1. It was used in the contrib-hls pr, I figure if there were issues it wouldn't have been ported over
2. The tests that use it seem to still pass.

I can undo it if you would prefer though.

Reply:
Nah, it's fine. Carry on :)